src/components/ProgressModal/ProgressModal.module.css

```diff
@@ -0,0 +1,34 @@
+@media only screen and (max-width: 768px) {
+  .loadingTextContainer {
+    margin-top: 10px;
+  }
+}
+
+@media only screen and (min-width: 769px) {
+  .content {
+    width: 350px;
+    height: 250px;
+  }
+
+  .loadingTextContainer {
+    margin-top: 40px;
+  }
+
+  .circularProgressContainer {
+    margin-top: 50px;
+  }
```

Comment:
I don't think loading describes what is happening good enough and then it will be much harder for the user to understand that '1/5' means how many of the delegations that's gone through. 'Processing delegations' describes what is happening much better.

Reply:
But I'll change the styling